pages/api/recipes/index.js

```diff
@@ -0,0 +1,15 @@
+import dbConnect from "@/db/connect";
+import Recipe from "@/db/models/Recipe";
+
+export default async function handler(request, response) {
+  try {
+    await dbConnect();
+    if (request.method === "GET") {
+      const recipe = await Recipe.find();
```

Comment:
in this route I would name the variable `recipes` because I expect them to be multiple.

Reply:
Done! Thank you.